JSQMessagesViewController/Views/MEGARecordView.swift

```diff
@@ -0,0 +1,69 @@
+import UIKit
+
+
+class MEGARecordView: UIView {
+    
+    var view: UIView!
+    @IBOutlet weak var timeLabel: UILabel!
+    @IBOutlet weak var volumeIcon: UIImageView!
+    
+    
+    // MARK: - init
+    override init(frame: CGRect) {
+        super.init(frame: CGRect.init(x: 0, y: 0, width: 180, height: 180))
```

Comment:
Is it a good idea for the creator of the class to set the frame?  `MEGARecordView(frame: CGRect(x: 0, y: 0, width: 180, height: 180))` or the frame is taken from XIB rather than hardcoding it in the `MEGARecordView` class itself. I feel this would make the class reusable in some other places. Let me know your thoughts.